src/daft-sql/src/modules/aggs.rs

```diff
@@ -78,6 +78,19 @@ impl SQLFunction for AggExpr {
 
 fn handle_count(inputs: &[FunctionArg], planner: &SQLPlanner) -> SQLPlannerResult<ExprRef> {
     Ok(match inputs {
+        [FunctionArg::Unnamed(FunctionArgExpr::Expr(SQLExpr::Value(Value::Number(s, _))))]
+            if s.as_str() == "1" =>
+        {
+            match &planner.current_plan {
+                Some(plan) => {
+                    let schema = plan.schema();
+                    unresolved_col(schema[0].name.clone())
+                        .count(daft_core::count_mode::CountMode::All)
+                        .alias("count")
+                }
+                None => unsupported_sql_err!("Wildcard is not supported in this context"),
+            }
+        }
```

Comment:
Nice, we should be supporting count with literals. Funny enough, SQL handles all literals the same here too, excpet the NULL value produces 0. I believe we have an expired slack thread about this fun stuff. So `count(2)` and `count(true)` etc. all the same!

**Schema (PostgreSQL v17)**

    CREATE TABLE t ( x INT );
    
    INSERT INTO t VALUES (1), (2), (3);

---

**Query #1**

    SELECT count(1), count(2), count(true), count(-1), count(null) FROM t;

| count | count | count | count | count |
| ----- | ----- | ----- | ----- | ----- |
| 3     | 3     | 3     | 3     | 0     |

---

[View on DB Fiddle](https://www.db-fiddle.com/f/8dURRozHxw5nsqh41YNWrF/0)